testutil/network/network.go

```diff
@@ -121,6 +121,8 @@ func newGenAccout(kr keyring.Keyring, name string, amount int64) (authtypes.Gene
 	balances := sdk.NewCoins(
 		sdk.NewCoin(fmt.Sprintf("%stoken", name), sdk.NewInt(amount)),
 		sdk.NewCoin(app.BondDenom, sdk.NewInt(amount)),
+		sdk.NewCoin(app.DisplayDenom, sdk.NewInt(amount)),
+		sdk.NewCoin(app.TokenDenom, sdk.NewInt(amount)),
```

Comment:
there are two entries for celes here? one upper case and one lower,  is that on purpose?

Reply:
right now, if you try to use celes or CELES for staking it won't work, you will only be able to transfer the token since the system relies on uceles

Reply:
yes, we should remove this before merging